.claude/agents/timo-codegen.md

```
@@ -0,0 +1,86 @@
+---
+name: timo-codegen
+description: Timo 프로젝트 코드 생성. DDD 레이어 구조, Java 21 관용 패턴, Timo 네이밍 컨벤션 준수. 완료 후 timo-git-committer → timo-code-reviewer 자동 실행.
+---
+
+# Timo Code Generator
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)